hq_superset/const.py

```diff
@@ -0,0 +1,2 @@
+# The name of the database for storing data related to CommCare HQ
+HQ_DATA = "HQ Data"
```

Comment:
nit: `CommCareHQ Data`?

Reply:
Reply nit: "CommCare HQ" [has a space in it](https://www.commcarehq.org/). :joy: 

Unfortunately this value needs to match the database name in existing environments. Previously, users had to create this database manually (see [instructions in README.md](https://github.com/dimagi/hq_superset/blob/master/README.md#create-a-postgres-database-connection-for-storing-hq-data), specifically, "The name of the DISPLAY NAME should be 'HQ Data' exactly". So we're stuck with "HQ Data" (even though the convention in Superset is to use a lowercase word, like "commcare-hq-data").